doc/en/skipping.rst

```diff
@@ -54,6 +54,15 @@ by calling the ``pytest.skip(reason)`` function:
         if not valid_config():
             pytest.skip("unsupported configuration")
 
+It is also possible to skip the whole module using
+``pytest.skip(reason, allow_module_level=True)`` at the module level:
+
+
+.. code-block:: python
+
```

Comment:
i do wonder if there is a better example for this (thats not looking like a skipif marker or a importorskip would already solve

Reply:
Maybe using the use case from previous discussion ? Something like:
```
if not pytest.config.getoption('--custom-flag'):
    pytest.skip('--custom-flag is missing, skipping tests', allow_module_level=True)
```